README.md

```diff
@@ -17,12 +17,56 @@ Webpack, Rollup, or Parcel.
 
-## Vocabularies and interoperability - solid-common-vocab-rdf
+## Vocabularies and interoperability
 
-[@inrupt/solid-common-vocab-rdf](https://github.com/inrupt/solid-common-vocab-rdf)
-allows developers to build interoperable apps by reusing well-known
-vocabularies. These libraries provide vocabularies available as constants that
-you just have to import.
+A fundamental requirement for any two systems to interoperate at all is that
+they have a shared understanding of the concepts and ideas relevant to those
+systems. That shared understanding is expressed as a common 'language', and
+the individual terms from that language are typically described and explained in
+dictionaries.
+
+The Solid eco-system uses RDF vocabularies (or slightly more formally,
+'ontologies') to describe concepts and ideas in a machine-readable, W3C
+standardized format (effectively, these RDF vocabularies act as dictionaries for
+systems to share concepts and ideas).
+
+There are already many common vocabularies published on the web, such as the
+[vCard vocabulary](https://www.w3.org/TR/vcard-rdf/) for describing the concepts
+and ideas associated with People and Organizations, or [Schema.org](https://schema.org/)
+for describing the things search engines are typically interested in, intended
+to be used by webmasters to mark up their pages in ways that those search
+engines can then understand.
+
+As a convenience for JavaScript developers, Inrupt publishes various NPM modules
+that provide JavaScript classes containing constants representing the individual
+terms described in many of these vocabularies today, including modules
+specifically for Solid-related vocabularies, and for Inrupt-specific
+vocabularies.
+
+By simply importing one of these NPM modules, developers have immediate access
+to all the terms described in all the RDF vocabularies referenced by that
+module.
+
+### Common RDF vocabularies
+
+This module bundles together JavaScript classes representing many of the most
+common RDF vocabularies published on the web today (e.g. FOAF, Schema.org,
+vCard, SKOS, etc.): [@inrupt/vocab-common-rdf](https://www.npmjs.com/package/@inrupt/vocab-common-rdf)
+
+### Solid-specific RDF vocabularies
+
+This module bundles together JavaScript classes representing all the RDF
+vocabularies related to Solid (e.g. Solid terms, WebACL, ACP, etc.): [@inrupt/vocab-solid-common](https://www.npmjs.com/package/@inrupt/vocab-solid-common)
+
+### Inrupt-specific RDF vocabularies
+
+This module bundles together JavaScript classes representing all the RDF
+vocabularies created and maintained by Inrupt (e.g. an Inrupt test vocabulary,
+Inrupt glossaries, Inrupt product vocabularies, etc.): [@inrupt/vocab-inrupt-common](https://www.npmjs.com/package/@inrupt/vocab-inrupt-common)
+
+**NOTE:** All the above NPM modules are currently generated and published
```

Comment:
(Spelling "NOTE" in all-caps will have that read out as an acronym by screen readers, I believe.)

```suggestion
**Note:** All the above npm modules are currently generated and published
```

Reply:
Sure thing, although by the same token won't having 'npm' in lowercase mean it won't be read out as an acronym by screen readers (meaning it'll become 'nipem', or something :) )...?!

Reply:
Yeah good point, blame npm and their stupid name capitalisation policy for that I suppose :)